### PR TITLE
feat(root): move docker images from github packages to aws ecr

### PR DESCRIPTION
**Pull request by merrcury**
### What changed? Why was the change needed?

This PR linked to below issues
- Fixes INF-444
- Fixes INF-446

### Screenshots
Please refer to AWS console or cli on dev region

### Special notes for your reviewer
PR #6754 slims these images down. So lets merge that first.

### CODE REVIEW

File: .github/workflows/cloud-deploy/cloud-deploy-webhook.yml [hunk 1]
```diff
@@ -16,6 +16,7 @@ jobs:
       health_check: true
```
Comment (SokratisVidros) [inline .github/workflows/cloud-deploy/cloud-deploy-webhook.yml]: You can safely delete all cloud-deploy ymls. They are not needed anymore.

**Review by SokratisVidros — APPROVED** [review]

File: .github/actions/docker/build-api/action.yml [hunk 2]
```diff
@@ -105,45 +80,43 @@ runs:
       id: build-image
       shell: bash
       env:
-        REGISTRY_OWNER: novuhq
-        DOCKER_NAME: ${{ inputs.docker_name }}
+        REGISTRY: ${{ steps.login-ecr.outputs.registry }}
```
Comment (SokratisVidros) [inline .github/actions/docker/build-api/action.yml:83]: Is this action reused during OS builds?

File: .github/workflows/cloud-deploy/cloud-deploy-webhook.yml (hunk 1 above)
Comment (merrcury) [reply .github/workflows/cloud-deploy/cloud-deploy-webhook.yml]: Sure 👍🏼

File: .github/actions/docker/build-api/action.yml (hunk 2 above)
Comment (merrcury) [reply .github/actions/docker/build-api/action.yml:83]: No, We have separate workflow built on top of matrix. Its entirely different from this 😄.

Comment (SokratisVidros) [reply .github/actions/docker/build-api/action.yml:83]: Do we maybe need to rename this shared YML part to make it clearer it's for the Novu Cloud?

Comment (merrcury) [reply .github/actions/docker/build-api/action.yml:83]: Sadly its only for dev 😢

**Pull request merged**